Classes/Sitegeist/MagicWand/Command/CloneCommandController.php

```diff
@@ -53,6 +59,17 @@ public function listCommand()
         }
     }
 
+    /**
+     * Clones the default preset
+     *
+     * @param boolean $yes confirm execution without further input
+     * @param boolean $keepDb skip dropping of database during sync
+     */
+    public function defaultCommand(bool $yes = false, bool $keepDb = false) : void
+    {
+        $this->presetCommand($this->defaultPreset, $yes, $keepDb);
```

Comment:
This is still missing a check wether a default preset exists

Reply:
In this case the `presetName` is null which results in a `The preset was not found!` error message.

But I make an explicit check.